libs/ui-lib/lib/ocm/components/clusterConfiguration/OcmClusterDetailsFormFields.tsx

```diff
@@ -109,6 +109,8 @@ export const OcmClusterDetailsFormFields = ({
     clusterExists,
     featureSupportLevelData,
   );
+  const [isDisabledHostsNetworkConfiguration, setIsDisabledHostsNetworkConfiguration] =
```

Comment:
Could you not use `values` from `useFormikContext` to retrieve the platform and act accordingly?
We shouldn't need an extra state.

Also please change the code in lines 90-92 to avoid doing an unnecessary call to `useFormikContext`.
Either use `values.openshiftVersion` or do something like

```
  const {  openshiftVersion  } = values;
```